app/src/controllers/administrate/sagas.js

```diff
@@ -37,10 +51,52 @@ function* fetchProjectData() {
   yield put(sectionDataAction(projectKey, isAdminAccess));
 }
 
+function* fetchActiveProjectData({ payload: { organizationSlug, projectSlug, projectKey } }) {
+  try {
+    yield put(fetchOrganizationBySlugAction(organizationSlug));
+
+    if (projectKey) {
+      yield put(setActiveProjectKeyAction(projectKey));
+      yield put(fetchProjectAction(projectKey));
+    } else {
+      yield take(createFetchPredicate(FETCH_ORGANIZATION_BY_SLUG));
+      yield put(fetchOrganizationProjectsAction(organizationSlug));
+      yield take(createFetchPredicate(FETCH_ORGANIZATION_PROJECTS));
+
+      const organizationProjects = yield select(organizationProjectsSelector);
+      const key = organizationProjects?.items?.find(({ slug }) => slug === projectSlug)?.key;
+
+      yield put(setActiveProjectKeyAction(key));
+      yield put(fetchProjectAction(key));
+      yield put(
+        fetchDashboardsAction({
+          [SIZE_KEY]: 300,
+        }),
```

Comment:
Why do we need to fetch dashboards here?
I suppose it should be removed.

Reply:
We have a new projectKey, if we don’t make a request, then there will be no dashboards for this project.